Content.Server/Abilities/Psionics/PsionicAbilitiesSystem.cs

```diff
@@ -203,8 +203,8 @@ public void MindBreak(EntityUid uid)
         RemComp<PsionicComponent>(uid);
         RemComp<InnatePsionicPowersComponent>(uid);
 
-        var ev = new OnMindbreakEvent();
-        RaiseLocalEvent(uid, ref ev);
+        // var ev = new OnMindbreakEvent();
+        // RaiseLocalEvent(uid, ref ev);
```

Comment:
```suggestion
        var ev = new OnMindbreakEvent();
        RaiseLocalEvent(uid, ref ev);
```
I'm very confident this should not be commented out